### PR TITLE
Semantic Interface Registry / ERC820 Extension

### PR DESCRIPTION
**Pull request by corydickson**
This is a first draft to add a `contexts` arrays that contain URI fields that point to semantic definitions inside contract ABIs.

### CODE REVIEW

File: EIPS/eip-x.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
+---
+eip: TBD
```
Comment (axic) [inline EIPS/eip-x.md]: Please use 1780.

File: EIPS/eip-x.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+title: JSON-LD Contract Metadata for Semantic Web Ontologies
+authors: Jordan Ellis <jelli@dorgtech.com>, Cory Dickson <cory.dickson94@gmail.com>
+type: ERC
+status: WIP
```
Comment (axic) [inline EIPS/eip-x.md]: There is no `WIP` status, please use `Draft`.

File: EIPS/eip-1780.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+category: Interface
+type: Standards Track
+status: Draft
+created: 2018-12-02
+---
+## Summary
+A standard interface for publishing and validating JSON/RDF/GraphQL Schema data models associated with ethereum addresses
+
+## Abstract
+Taking some of the concepts in [EIP-1357](https://github.com/eth-registry/EIPs/blob/EIP-1357-Address-Metadata/EIPS/eip-1456.md) but not including the fields illustrated in their Specification:
```
Comment (axic) [inline EIPS/eip-1780.md]: There doesn't seem to be an EIP-1357. #1357 is something unrelated.

Comment (axic) [reply EIPS/eip-1780.md]: It seems you are referring to #1456.

**Comment by corydickson** [comment]
Working on an updated revision before EOY with test cases. Thank you for the patience.

**Comment by corydickson** [comment]
ABI supports multiple named return parameters, which was needed to continue; just have been busy with other projects https://github.com/ethereum/solidity/pull/7534

**Pull request closed without merging**